### ISSUE TITLE
Stay on server

### ISSUE THREAD

**Issue by Salvationdk** [ermiyaeskandary/Slither.io-bot#187]
Would like to have the ability to stay on the connected server instead of reloading to new server ip after death.

**Comment by ermiyaeskandary** [comment]
If you use ESC , it will stay on the same server.

**Comment by Salvationdk** [comment]
yes i know,  sometimes the server change ip on reload :) So a force IP would be cool

**Comment by ROacktree** [comment]
https://github.com/ErmiyaEskandary/Slither.io-bot/issues/216 

Provides a way to do more than remain on the current server.  As suggested by the code labeling it allows one to meet up with friends.

**Comment by Seple** [comment]
hmmm... 
Development - MENU "TOP" OPTION ?
List "TOP" Results point + PLUS -> Displaying time play.
What do you think ?

**Comment by grovtor** [comment]
yes, I would enjoy this option
